src/items/Camembert.java

```diff
@@ -10,12 +11,25 @@
  */
 public class Camembert extends Item {
     /**
+     * A Camembert használata. Eltávolítja a játékos tárgyai közül a Camembert-et, 
+     * és a szobához hozzáad egy PoisonEffect-et, és azt aktiválni kell.
      * @param room
      * @param player
      */
     @Override
-    void use(Room room, Player player) {
+    public void use(Room room, Player player) {
 
+        System.out.println("Camembert.use(Room, Player)");
+
+        player.removeItem(this);
+
+        PoisonEffect camembertEffect = new PoisonEffect();
```

Comment:
Az effektnek tudnia kell az itemről ami tulajdonolja, előtte nem szűntethető meg. Az effekt lejárása hívja meg a tárgy törlését

Reply:
Effect(Item givenBy) {
        this.givenBy = givenBy;
    }

Ez lesz majd a konstruktor az effekt osztályhoz

Reply:
Hozzáadtam az Effect konstruktorához a tárgyat és az időtartamot. (Effect(givenBy, duration))